ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+
+_Detailed issue description here. More information at [https://flutter.io/bug_reports/](https://flutter.io/bug_reports/)_
```

Comment:
Eh, no. Things got backed up over the past couple of days. Do you want me to separate this into another PR?